flutter-intellij.iml

```diff
@@ -80,12 +80,12 @@
     <orderEntry type="library" name="Dart Packages" level="project" />
     <orderEntry type="module" module-name="intellij.platform.externalSystem" />
     <orderEntry type="module" module-name="intellij.gradle.common" />
-    <orderEntry type="library" name="com.android.tools:common" level="project" />
-    <orderEntry type="library" name="studio-analytics-proto" level="project" />
-    <orderEntry type="library" name="protobuf" level="project" />
     <orderEntry type="module" module-name="intellij.java.compiler.impl" />
     <orderEntry type="module" module-name="plugin" />
     <orderEntry type="module" module-name="intellij.platform.serviceContainer" />
+    <orderEntry type="library" name="com.android.tools:common" level="project" />
```

Comment:
Normally, I'd remind someone that changes to this file also need to be made to the other .iml file, but this is basically a no-op.

Reply:
ah, happy to (this was just a manual edit to improve the readability slightly)